internal/config/config.go

```diff
@@ -81,6 +81,8 @@ type Config struct {
 	AutoInstrumentationJavaImage string `yaml:"auto-instrumentation-java-image"`
 	// OpenshiftCreateDashboard creates an OpenShift dashboard for monitoring the OpenTelemetryCollector instances
 	OpenshiftCreateDashboard bool `yaml:"openshift-create-dashboard"`
+	// CreateNetworkPolicies creates network policies for the operator and operands created by the operator.
```

Comment:
```suggestion
	// CreateNetworkPolicies creates network policies for the operator created by the operator.
```
When adding the support for the Operands, should you be able to disable it on the CR itself?